furya/chain.json

```diff
@@ -29,9 +29,9 @@
   },
   "codebase": {
     "git_repo": "https://github.com/furysport/furya-chain",
-    "recommended_version": "v1.0.0",
+    "recommended_version": "v2.0.0",
```

Comment:
Please add the new version object in the `versions` array as well (while keeping v1 before it)

Reply:
As per your instruction, I have the objects updated in the latest commit.
I've followed the same convention as the other chains. Please let me know if this requires further revision.